chainerrl/replay_buffer.py

```diff
@@ -160,7 +170,9 @@ def load(self, filename):
                 self.memory, maxlen=self.memory.maxlen)
 
     def stop_current_episode(self):
-        pass
+        if 0 < len(self.last_n_transitions) < self.num_steps:
```

Comment:
Isn't is possible that `len(self.last_n_transitions) == self.num_steps`?

Reply:
Yes, but then `self.last_n_transitions` would have been added to the memory already, and we wouldn't want to do it twice. This catches the edge case that the episode ends in less than `self.num_steps`, and nothing has been added to the memory. The important part of this method is that `self.last_n_transitions` is cleared, so that we don't merge transitions from the next episode to this one.

Reply:
Ah I see. So `len(self.last_n_transitions) < self.num_steps` is correct.